Localhost/Mac/System/telemetry/installer/com.logzio.OTELCollector.plist

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+  <dict>
+
+    <key>Label</key>
+    <string>com.logzio.OTELCollector</string>
+
+    <key>RunAtLoad</key>
+    <true/>
+
+    <key>StandardErrorPath</key>
+    <string>DIR/otelcol-contrib_stderr.log</string>
```

Comment:
Please update the installation directory to /opt

Reply:
done